cf_units/__init__.py

```diff
@@ -484,17 +394,17 @@ def _discard_microsecond(date):
     dates = np.asarray(date)
     shape = dates.shape
     dates = dates.ravel()
-    # Create date objects of the same type returned by utime.num2date()
+    # Create date objects of the same type returned by cftime.num2date()
     # (either datetime.datetime or cftime.datetime), discarding the
     # microseconds
     dates = np.array([d and d.__class__(d.year, d.month, d.day,
                                         d.hour, d.minute, d.second)
                       for d in dates])
-    result = dates[0] if shape is () else dates.reshape(shape)
+    result = dates[0] if shape == () else dates.reshape(shape)
```

Comment:
These were giving me SyntaxWarnings (python3.9) so I thought I might as well change them while I'm in there.